.circleci/config.yml

```diff
@@ -13,20 +13,8 @@ jobs:
     executor: go
     steps:
       - checkout
-
-      # Restore go module cache if there is one
-      - restore_cache:
-          keys:
-            - memberlist-modcache-v1-{{ checksum "go.mod" }}
```

Comment:
With the introduction of go modules it is actually faster to download just the module from the go proxy than to try and cache the local `/go/pkg/mod`.